roles/comanage-registry/defaults/main.yml

```diff
@@ -88,7 +90,25 @@ comanage_mailers:
     #charset: "utf-8"
     #header_charset: "utf-8"
 
-# Directory containing public static pages to copy
-#comanage_public_pages_dir: "{{ inventory_dir }}/public"
-
+# Used in login.php and logout.php
 comanage_session_name: "CAKEPHP"
+comanage_core_session_conf:
+  defaults: "php"
+
+# Dis/Enable COmanage cache
+#comanage_core_cache: true
+# Dis/Enable COmanage debug mode
+#comanage_core_debug: 1
+
+#comanage_logout_location: "/registry.sso/Logout?return=%2Fregistry%2Fpages%2Fpublic%2Floggedout"
+# Used in Controller/AppController.php, View/Layout/default.ctp, View/Layout/lightbox.ctp
+#comanage_timezone_cookie_name: "cookie_tz_auto"
+
+#comanage_memcache_hosts:
+#  - ip: '127.0.0.1'
+#    port: 11211
+#  - ip: '127.0.0.1'
+#    port: 11212
+
+# Backup the Comanage directory
```

Comment:
Where is the backup maintained if this is set to true and is there a specific naming scheme?